.github/workflows/main.yml

```
@@ -313,10 +313,19 @@ jobs:
       # the coverage reports out of the container and into the host filesystem.
       - name: Copy coverage reports to the host
         run: ./tools/copy_docker_coverage_reports.sh test_container
-      - name: Upload coverage reports to Codecov
+      - name: Upload Main coverage report to Codecov
```

Nit, matching the name of the flag:

```suggestion
      - name: Upload JavaScript coverage report to Codecov
```